src/pages/index.js

```diff
@@ -36,15 +43,26 @@ const addCardPopup = () => {
 };
 
 const showImageModal = (card) => {
-  imagePopup.open(card.getImageLink(), card.getName());
+  imagePopup.open(card.getCardData());
+};
+
+const deleteCardClick = (card) => {
+  cardDeletePopup.setInputValues(card.getCardData());
+  cardToDelete = card;
+  cardDeletePopup.open();
+};
+
+const changeProfileAvatar = (userInfo) => {
+  changeAvatarPopup.setInputValues(userInfo.getUserInfo());
```

Comment:
**Could be improved**

Please, call `resetValidation` everywhere to clear possible errors from the inputs

Reply:
done